### PR TITLE
Add quick-look example

### PR DESCRIPTION
**Pull request by robatron**
Adds a quick reference example at the top of the code style guide.

### CODE REVIEW

File: docs/javascript.md [hunk 1]
```diff
@@ -2,11 +2,58 @@
...
+Here's a quick example of JavaScript formatted following this code style guide:
 
+```js
+'use strict';
+
+function foo ( x, y, z ) {
+    bar( 1, b );
+
+    var lastName;
+    var i = 0;
```
Comment (agats) [inline docs/javascript.md]: Single letter names, https://github.com/rei/js-style-guide#naming-conventions

Comment (robatron) [inline docs/javascript.md]: Good catch. I'll update. $todo

**Comment by agats** [comment]
Looks good, it's a nice snippet of code that follows the JS style guide.

**Comment by agats** [comment]
+1

**Comment by petersgrandadventure** [comment]
+1

**Comment by robatron** [comment]
Two +1 approvals. Merging.

**Pull request merged**